components/recurring-contributions/UpdateOrderPopUp.js

```diff
@@ -223,6 +223,7 @@ const UpdateOrderPopUp = ({ setMenuState, contribution, setShowPopup }) => {
                   {checked && flexible ? (
                     <Fragment>
                       <div
+                        role="presentation"
```

Comment:
I don't get this one?

Reply:
@Betree : The `div` was added only to prevent the `select` menu from closing immediately. Other than that there's no value in the `div` which is why I added the `presentation` role. 😄 If not ideally this `div` needs to have a keyboard event (a11y rule, `jsx-a11y/click-events-have-key-events`).

Reply:
But `role=presentation` has [a specific meaning](https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/presentation/PresentationRoleExamples.html), that does not seem to match what we're doing here:
> role="presentation" hides the image from the accessibility API and does not publish the alt attribute contents.

I'm afraid that this could make the entire select inaccessible to screen readers or other accessibility tools. We'd better ignore the warning if we're not sure about this one.

Reply:
@Betree : I was going with the following definition where it says that inner elements are not hidden and seems to be available for accessibility tools. This seems to be the difference between presentation role and `aria-hidden`. 

https://www.w3.org/TR/wai-aria-practices-1.1/#presentation_role_effects

Anyways I think it also makes sense to add an exception to this (to ignore the warning) to make it doubly sure; so I've went ahead and did that. 😄

Reply:
You're actually right that it should not impact accessibility. Disabling the warning is a good approach for now, let's keep your changes. Thanks :-)